src/index.html

```diff
@@ -11,7 +11,108 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post">
+      <fieldset class="formgroup">
+        <legend>Personal information</legend>
+        <label for="surname" class="forminput">Surname:
+          <input type="text" name="surname"
+                 id="surname" autocomplete="off"
+          >
+        </label>
+        <label for="name" class="forminput">Name:
+          <input type="text" name="name"
```

Comment:
please, apply formatting mentioned in the check list
```suggestion
          <input 
            type="text" 
            name="name"
```

![Screenshot 2023-04-07 at 17 47 36](https://user-images.githubusercontent.com/6526149/230628654-6e629403-7a73-4bad-b16a-6dfa81dda021.png)